react/ui-components/src/hoc/app-config-screen-builder/AppFieldComposer.js

```
@@ -0,0 +1,1173 @@
+import React, { Fragment, useEffect, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { PRIMARY_COLOR } from "./app-config-utils/constants";
+import { DustbinIconNew as DustbinIcon } from "../../atoms/CustomSVG";
+import { useAppConfigContext } from "./AppConfigurationWrapper";
+import { InfoOutline } from "@egovernments/digit-ui-svg-components";
+
+const CheckBoxes = ({ t, option, optionKey, isLabelFirst }) => {
+  return (
+    <div>
+      {option?.map((item, index) => (
+        <CheckBox onChange={(e) => {}} value={""} label={t(`${item?.[optionKey]}`)} isLabelFirst={isLabelFirst} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)